docs/standalone-binary.md

```
@@ -0,0 +1,119 @@
+
+Locally:
+
+```sh
+bun run build:standalone           # release → build/release-standalone/bun-standalone
+bun run build:standalone:debug     # debug   → build/debug-standalone/bun-standalone-debug
+```
+
+In CI, each release platform gets two extra steps that reuse the existing
+`build-cpp` artifact:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)